src/components/atoms/Header.styles.tsx

```diff
@@ -9,6 +10,8 @@ export const HeaderContainer = styled.header`
   top: 0;
```

Comment:
why ?

Reply:
Without top: 0, the sticky property does not fix the header at the top of the viewport